lib/widgets/playlists_list_item.dart

```diff
@@ -26,7 +32,7 @@ class PlaylistsListItem extends StatelessWidget {
       child: Material(
         type: MaterialType.transparency,
         child: InkWell(
-          onTap: () {},
+          onTap: onTap ?? () {},
```

Comment:
Don't default to `(){}`, this will add an inkwell ripple even though it is not actually tappable